<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simple Text Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTextEdit" name="plainTextEdit"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_new">
          <property name="text">
           <string>New</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_open">
          <property name="text">
           <string>Open</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_save">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_help">
          <property name="text">
           <string>Help</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_short">
          <property name="text">
           <string>Shortcuts</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_exit">
          <property name="text">
           <string>Exit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_lang">
          <property name="placeholderText">
           <string>Language</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_print">
          <property name="text">
           <string>Print</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>Language</string>
     </property>
     <addaction name="actionEnglish"/>
     <addaction name="actionRussian"/>
    </widget>
    <widget class="QMenu" name="menuStyleSheet">
     <property name="title">
      <string>StyleSheets</string>
     </property>
     <addaction name="actionDark"/>
     <addaction name="actionDefault"/>
    </widget>
    <widget class="QMenu" name="menuFormat">
     <property name="title">
      <string>Format</string>
     </property>
     <widget class="QMenu" name="menuAlignment">
      <property name="title">
       <string>Alignment</string>
      </property>
      <addaction name="actionLeft"/>
      <addaction name="actionRight"/>
      <addaction name="actionCenter"/>
     </widget>
     <addaction name="actionGet_font"/>
     <addaction name="actionSet_font"/>
     <addaction name="actionRandomize_font"/>
     <addaction name="actionChange_font"/>
     <addaction name="separator"/>
     <addaction name="menuAlignment"/>
    </widget>
    <addaction name="actionShortcuts"/>
    <addaction name="menuLanguage"/>
    <addaction name="menuStyleSheet"/>
    <addaction name="separator"/>
    <addaction name="menuFormat"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionContext_Help"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionShortcuts">
   <property name="text">
    <string>Shortcuts</string>
   </property>
  </action>
  <action name="actionContext_Help">
   <property name="text">
    <string>Context Help</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionRussian">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Русский</string>
   </property>
  </action>
  <action name="actionDark">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
  <action name="actionDefault">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Default</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionGet_font">
   <property name="text">
    <string>Get font</string>
   </property>
  </action>
  <action name="actionSet_font">
   <property name="text">
    <string>Set font</string>
   </property>
  </action>
  <action name="actionRandomize_font">
   <property name="text">
    <string>Randomize font</string>
   </property>
  </action>
  <action name="actionLeft">
   <property name="text">
    <string>Left</string>
   </property>
  </action>
  <action name="actionRight">
   <property name="text">
    <string>Right</string>
   </property>
  </action>
  <action name="actionCenter">
   <property name="text">
    <string>Center</string>
   </property>
  </action>
  <action name="actionChange_font">
   <property name="text">
    <string>Change font ...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
